cli/command/node/builders_test.go

```diff
@@ -0,0 +1,149 @@
+package node
```

Comment:
This node builder would probably be useful across a few packages. What do you think about moving it to a new package and exporting it?